source/common/network/tcp_listener_impl.cc

```diff
@@ -90,18 +91,24 @@ void TcpListenerImpl::onSocketEvent(short flags) {
     cb_.onAccept(
         std::make_unique<AcceptedSocketImpl>(std::move(io_handle), local_address, remote_address));
   }
+
+  ENVOY_LOG_MISC(trace, "TcpListener {} accepted {} new connections.",
+                 socket_->ioHandle().localAddress()->asStringView(), connections_accepted_count);
```

Comment:
I think the value `connections_accepted_count` may also include rejected connections, and not only accepted.